pkg/util/perf/perf_linux.go

```diff
@@ -30,15 +31,18 @@ import (
 )
 
 type PerfCollector struct {
-	CgroupFd          int
+	cgroupFile        *os.File
```

Comment:
do we need to store both the fd and the file pointer

Reply:
At first I store both for minimum code modification, but I think it's okay to abort usage of cgroupfd. Will remove it.